Jenkinsfile

```diff
@@ -53,15 +53,17 @@ pipeline {
             }
             steps {
                 script {
-                    if (env.BRANCH_NAME == 'master') {
+                    if (env.BRANCH_NAME == 'aws_wps_ecr_pipeline') {
```

Comment:
`aws_wps_ecr_pipeline` is the new master?

Reply:
Sharp, testing off this branch ;-)